hadoop-pipeline-application/src/main/java/org/molgenis/hadoop/pipeline/application/HadoopPipelineApplication.java

```diff
@@ -44,12 +56,50 @@ public static void main(String[] args)
 	public int run(String[] args) throws IOException, ParseException, ClassNotFoundException, InterruptedException
 	{
 		Configuration conf = getConf();
+
+		// YARN fix for: java.lang.OutOfMemoryError: Java heap space
+		conf.set("mapreduce.map.memory.mb", "2048");
+		conf.set("mapreduce.map.java.opts", "-Xmx1024m");
+
```

Comment:
According to https://hadoop.apache.org/docs/r1.0.4/commands_manual.html#Overview `-D <property=value>` should already work as the class `implements Tool`, though when tested on a local psuedo-cluster it seemed to not differ (while this fix did). Perhaps using `org.molgenis.hadoop.pipeline.application.inputdigestion.CommandLineParser` somehow disables it, though don't know for sure.

I agree that a command line implementation is prefered (though it's more a question of when and whether to first look to find the issue with the `Tools` version not working or simply right away implement it into the InputParser from this application directly).